frontend/public/components/deployment.tsx

```diff
@@ -91,7 +92,18 @@ const DeploymentDetails: React.FC<DeploymentDetailsProps> = ({obj: deployment})
     <div className="co-m-pane__body">
       <SectionHeading text="Deployment Overview" />
       {deployment.spec.paused && <WorkloadPausedAlert obj={deployment} model={DeploymentModel} />}
-      <DeploymentPodCounts resource={deployment} resourceKind={DeploymentModel} />
+      <TopologyDataController
+        namespace={deployment.metadata.namespace}
+        application={deployment.metadata.labels['app.kubernetes.io/part-of']}
```

Comment:
Causes an error in some cases because we actually don't need the application to display the pod status.
```
react-dom.development.js:20312 Uncaught TypeError: Cannot read property 'app.kubernetes.io/part-of' of undefined
    at DeploymentOverviewDetails (deployment-overview.tsx:24)
    at renderWithHooks (react-dom.development.js:12938)
    at updateFunctionComponent (react-dom.development.js:14627)
    at beginWork (react-dom.development.js:15637)
    at performUnitOfWork (react-dom.development.js:19312)
    at workLoop (react-dom.development.js:19352)
    at renderRoot (react-dom.development.js:19435)
    at performWorkOnRoot (react-dom.development.js:20342)
    at performWork (react-dom.development.js:20254)
    at performSyncWork (react-dom.development.js:20228)
```

Reply:
This issue has been resolved as I am not using the application anymore.